### PR TITLE
Delightful 404 page

### LINKED ISSUE #65: Slick 404 page
Perhaps with an automatic search based on the url.

Should just live at `/404.html`.

### PR DESCRIPTION
**Pull request by benbalter**
Took a first pass at fleshing out a delightful 404 page to replace the current pages default 404.

Dropped in the google 404 widget to hopefully provide some help locating content for now, but may be able to leverage the search API to do something slick here in the long run.

Thoughts on something charming / witty / fun?

screen shot 2013-10-16 at 12 30 46 pm

Fixes #65.

### CODE REVIEW

File: 404.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Four oh four
+---
+
+## Looks like the page you requested wasn't found :(
+
+<script type="text/javascript">
+  var GOOG_FIXURL_LANG = 'en';
+  var GOOG_FIXURL_SITE = 'http://government.github.com
```
Comment (parkr) [inline 404.md]: Forgot a `'` here :)

File: 404.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+layout: page
+title: Four oh four
```
Comment (parkr) [inline 404.md]: I'd go with the traditional `not found` here :)

File: 404.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+---
+
+## Looks like the page you requested wasn't found :(
```
Comment (parkr) [inline 404.md]: `The page you're looking for doesn't seem to be there! Maybe we can find it together if you search below.`

**Comment by parkr** [comment]
[Vimeo does a great job with their 404](http://vimeo.com/not-found), suggesting spontaneous discovery of Vimeo and its content.

**Comment by benbalter** [comment]
@parkr great suggestions all around.

Merging so that we have something, but feel free to weigh in here or on an new issue... would love to continue to iterate.

**Pull request merged**